### DISCUSSION TITLE
redux peer dependency in appui-react

### DISCUSSION THREAD

**Discussion by AAugustBentley** [iTwin/appui#1099] [Q&A] [answered]
We started moving from `redux` to `zustand` and other storing options (which `appui` seems to be doing as well) but `redux` is still added to `peer dependencies` and we have to install in our projects to be able to use `appui-react`.

Is this a stale peer dep and could it be removed?
![image](https://github.com/user-attachments/assets/59b7811c-3440-4cdd-8ccd-cd147318dbc1)
![image](https://github.com/user-attachments/assets/34bf0a61-28f3-4923-bafb-9219987dc683)

**Comment by GerardasB** [comment, marked as answer]
This dependency will be removed with AppUI `6.0.0` due to our API support policy https://www.itwinjs.org/learning/api-support-policies/#api-deprecation-policy
API deprecation notes https://github.com/iTwin/appui/releases/tag/release%2F4.15.0
